services/manifest-repo-export-service/pkg/repository/repository.go

```diff
@@ -1279,6 +1280,47 @@ func (s *State) FixReleasesTimestamp(ctx context.Context, transaction *sql.Tx, a
 	return nil
 }
 
+func(r *repository) FixCommitsTimestamp(ctx context.Context, state State) (error) {
+	revwalk, err := r.repository.Walk()
+	if err != nil {
+		return fmt.Errorf("failed to create revwalk: %v", err)
+	}
+	head, err := r.repository.Head()
+	if err != nil {
+		return fmt.Errorf("Failed to get HEAD: %v", err)
+	}
+
+	// Push HEAD to revwalk
+	err = revwalk.Push(head.Target())
+	if err != nil {
+		return fmt.Errorf("Failed to push HEAD to revwalk: %v", err)
+	}
+	dbHandler := state.DBHandler
+	err = dbHandler.WithTransaction(ctx, false, func(ctx context.Context, transaction *sql.Tx) error {
+		err = revwalk.Iterate(func(commit *git.Commit) bool {
+			commit, err := r.repository.LookupCommit(commit.Id())
+			if err != nil {
+				logger.FromContext(ctx).Sugar().Errorf("failed to lookup commit %s: %v", commit.Id().String(), err)
+				return true // continue
+			}
+
+			logger.FromContext(ctx).Sugar().Infof("Commit: %s, Time: %s\n", commit.Id().String(), time.Unix(commit.Committer().When.Unix(), 0))
+			err = dbHandler.DBUpdateCommitTransactionTimestamp(ctx, transaction, commit.Id().String(), commit.Committer().When)
+			if err != nil {
+				logger.FromContext(ctx).Sugar().Errorf("failed to lookup commit %s: %v", commit.Id().String(), err)
+				return true
+			}
+			
+			return true
+			})
+		return err
+	})
+	if err != nil {
+		logger.FromContext(ctx).Sugar().Errorf("failed during revwalk: %v", err)
+	}
+	return nil
```

Comment:
this should return the error 
```suggestion
	if err != nil {
		return fmt.Errorf("failed during revwalk: %v", err)
	}
	return nil
```